clang/test/Driver/time.c

```diff
@@ -0,0 +1,24 @@
+// The -time option prints timing information for the various subcommands in a
+// format similar to that used by gcc. When compiling and linking, this will
+// include the time to call clang-${LLVM_VERSION_MAJOR} and the linker. Since
+// the name of the linker could vary across platforms, and name of the compiler
+// could be something different, just check that whatever is printed to stderr
+// looks like timing information.
+
+// RUN: %clang -time -c -O3 -o /dev/null %s 2>&1 \
```

Comment:
This requires a hard-coded target triple like `--target=x86_64` and `REQUIRES: x86-registered-target`.

Delete unneeded -O3.

Reply:
Wouldn't this restrict this test to running only when that target has been enabled? Is there a way to set the target to some target that has been enabled in that build? I would like to avoid restricting where the test runs. Perhaps I am missing something.

Reply:
@MaskRay, am I right in understanding that the changes that you have requested would restrict the test to run only on x86? If not, is there somewhere this is explained so I understand this better. Thanks.

Reply:
If the default target is certain niche OSes AIX, z/OS , the output could be very different. While we are comfortable with Linux on many architectures, certain architectures could have strange behaviors (MIPS). So just stick with x86_64-linux

Reply:
Thanks for the explanation. I have restricted the tests to only run on `x86_64-linux` with a comment explaining the reasoning for the restriction.